platforms/minigame/platforms/taobao-mini-game/wrapper/builtin/MultiWebSocket.js

```diff
@@ -1,42 +1,45 @@
 const _utils = require('../utils');
 
 export default class MultiWebSocket {
-  static CONNECTING = 0 // The connection is not yet open.
-  static OPEN = 1 // The connection is open and ready to communicate.
-  static CLOSING = 2 // The connection is in the process of closing.
-  static CLOSED = 3 // The connection is closed or couldn't be opened.
+  static CONNECTING = 0; // The connection is not yet open.
+  static OPEN = 1; // The connection is open and ready to communicate.
+  static CLOSING = 2; // The connection is in the process of closing.
+  static CLOSED = 3; // The connection is closed or couldn't be opened.
 
-  binaryType = '' // TODO: update binaryType
-  bufferedAmount = 0 // TODO: update bufferedAmount
-  extensions = ''
+  binaryType = ''; // TODO: update binaryType
+  bufferedAmount = 0; // TODO: update bufferedAmount
+  extensions = '';
 
-  onclose = null
-  onerror = null
-  onmessage = null
-  onopen = null
+  onclose = null;
+  onerror = null;
+  onmessage = null;
+  onopen = null;
 
-  protocol = '' // TODO: It is not currently available within the applet, but actually needs to be returned according to the sub-protocol selected by the server
-  readyState = 3
+  protocol = ''; // TODO: It is not currently available within the applet, but actually needs to be returned according to the sub-protocol selected by the server
+  readyState = 3;
 
   constructor(url, protocols = []) {
     if (typeof url !== 'string' || !(/(^ws:\/\/)|(^wss:\/\/)/).test(url)) {
-      throw new TypeError(`Failed to construct 'WebSocket': The URL '${url}' is invalid`)
+      throw new TypeError(`Failed to construct 'WebSocket': The URL '${url}' is invalid`);
     }
 
-    this.url = url
-    this.readyState = MultiWebSocket.CONNECTING
+    this.url = url;
+    this.readyState = MultiWebSocket.CONNECTING;
 
     const socketTask = my.connectSocket({
       url,
-      multiple: true
+      multiple: true,
+      fail: (res) => {
+        console.warn("MultiWebSocket connectSocket fail",res);
```

Comment:
When connectSocket fails, the callback registered via socketTask.onError should be invoked. During the adaptation process, the Taobao team confirmed that onError would be triggered upon connect failure. Is the addition of the fail callback here specifically because onError isn’t being called?